integrations/Drip/browser.js

```diff
@@ -0,0 +1,151 @@
+import get from "get-value";
+import logger from "../../utils/logUtil";
+import { removeUndefinedAndNullValues } from "../utils/commonUtils";
+
+class Drip {
+  constructor(config) {
+    this.accountId = config.accountId;
+    this.campaignId = config.campaignId;
+    this.name = "DRIP";
+  }
+
+  init() {
+    logger.debug("===In init Drip===");
+
+    var _dcq = _dcq || [];
+    var _dcs = _dcs || {};
+    _dcs.account = this.accountId;
+
+    (function () {
+      var dc = document.createElement("script");
+      dc.type = "text/javascript";
+      dc.async = true;
+      dc.src = `//tag.getdrip.com/${this.accountId}.js`;
+      var s = document.getElementsByTagName("script")[0];
+      s.parentNode.insertBefore(dc, s);
+    })();
+  }
+
+  identify(rudderElement) {
+    logger.debug("===In Drip identify===");
+
+    const { message } = rudderElement;
+    if (!message.context.traits) {
+      logger.error("user traits not present");
+      return;
+    }
+
+    let payload = {
+      email: get(message, "context.traits.email"),
+      new_email: get(message, "context.traits.new_email"),
+      user_id: get(message, "user_id"),
+      tags: get(message, "context.traits.tags"),
+      remove_tags: get(message, "context.traits.remove_tags"),
+      prospect: get(message, "context.traits.prospect"),
+      eu_consent: get(message, "context.traits.eu_consent"),
```

Comment:
`eu_consent` can have two string values as per the doc. maybe add a check, if its something invalid, dont include in the payload.